src/lib/actions/sources.ts

```diff
@@ -4,30 +4,30 @@ import { createSource, deleteSource, updateSource } from "@/lib/db/sources";
 import { redirect } from "next/navigation";
 import { z } from "zod";
 
-const createSourceFormDataSchema = z.object({
+const CreateSourceFormDataSchema = z.object({
```

Comment:
What does the change to PascalCase indicate here?

Reply:
That it's an action I guess?

Reply:
This is something I go back and forth with, and so does the community, from what I can tell. It's just a personal preference thing. It's not doing anything special. I just felt like saying "maybe we should capitalize zod schemas" because I keep seeing examples where they are capitalized. I guess it feels like they stand out better to me.